js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -40,8 +40,8 @@ export default class TypedInput extends Component {
       PropTypes.string
     ]).isRequired,
 
-    accounts: PropTypes.object,
     allowCopy: PropTypes.bool,
+    allowedValues: PropTypes.array,
```

Comment:
DeployContract also used `TypedInput`, don't see changes addressing the changed interface? (As well as Contract queries, Mining Settings - all passing accounts instead of allowedValues to TypedInput?)

Reply:
It's not a required prop. If empty, it should be ignored (ie. all values are allowed)

Reply:
Perfect, however we are still passing `accounts` through at various places now and assuming (from the caller's perspective) it does get used, otherwise the caller shouldn't provide it. 

If those are needed, should be replaced with the real values, if not, should be removed since it creates confusion in the codebase. (And warnings when running)

Reply:
Right.

Reply:
Fixed